File: app/src/main/java/com/polok/eubmanagement/presentation/dashboard/DashboardActivity.java
package com.polok.eubmanagement.presentation.dashboard;

import android.os.Bundle;
import androidx.navigation.Navigation;
import androidx.navigation.ui.NavigationUI;
import com.polok.eubmanagement.R;
import com.polok.eubmanagement.base.BaseActivity;
import com.polok.eubmanagement.databinding.ActivityDashboardBinding;

public class DashboardActivity extends BaseActivity<ActivityDashboardBinding> {
    @Override
    protected ActivityDashboardBinding initViewBinding() {
        return ActivityDashboardBinding.inflate(getLayoutInflater());
    }

    @Override
    protected void initOnCreateView(Bundle savedInstanceState) {
        NavigationUI.setupWithNavController(
                binding.bottomNav,
                Navigation.findNavController(this, R.id.fragment_container)
        );
    }
}

File: app/src/main/java/com/polok/eubmanagement/presentation/dashboard/DashboardActivity.kt
package com.polok.eubmanagement.presentation.dashboard

import android.os.Bundle
import androidx.navigation.Navigation.findNavController
import androidx.navigation.ui.NavigationUI.setupWithNavController
import com.polok.eubmanagement.R
import com.polok.eubmanagement.base.BaseActivity
import com.polok.eubmanagement.databinding.ActivityDashboardBinding

class DashboardActivity : BaseActivity<ActivityDashboardBinding>(
    viewBindingFactory = ActivityDashboardBinding::inflate
){
    override fun initOnCreate(savedInstanceState: Bundle?) {

        setupWithNavController(
            binding.bottomNav, findNavController(this, R.id.fragment_container)
        )
    }
}